packages/commercetools-sdk-middleware-auth/README.md

```diff
@@ -0,0 +1,39 @@
+# commercetools-sdk-middlware-auth
+
+Auth middlewares collection for usage with `@commercetools/sdk`
```

Comment:
Yep, like @Siilwyn said. The "node" part is redundant since it's a node module per-se ;)